src/image-sync-manager.ts

```diff
@@ -47,15 +56,25 @@ export class ImageSyncManager {
   };
 
   private _onTimedMetadata = ({payload}: TimedMetadata) => {
-    const activeSlide = payload.cues?.find(cue => {
-      return cue.track?.label === cuepoint.CUE_POINTS_TEXT_TRACK && cue.value?.data?.cuePointType === ThumbCuePointType;
-    });
-    this._imagePlayer.setActive(activeSlide ? activeSlide.value.data.id : null);
+    if (payload.label === KalturaPlayer.cuepoint.CUE_POINTS_TEXT_TRACK) {
```

Comment:
label data shouldn't come from core player.
The fact that the data is coming from a text track with a label is an implementation detail, and the data source for timed metadata could be other then texttrack.
The data should be filtered from the cue data and from the internal data source type if possible.

Reply:
use-case: for live entries id3 tag pushes to text-track and time to time trigger metadata_changed event with empty array of cues (that means no one id3 cue-point active). In the same time image-player also listen metadata_changed and should remove slide if no-one active slide cue-points currently active.

issue: subscribers (in our case image-player) to metadata doesn't know which text-track emitted empty data.

solutions:
1) use label of TextTrack in metadata_changed to filter-out inconsistent data (current solution);
2) each time on metadata_changed event handled all the subscribers should get all active cue-points and make a search if necessary type of cue-points exist there.

Reply:
for now "TIMED_METADATA" event trigger getting active cue-points from cue-point manager.
Once we implement support of single "metadata" TextTrack in cue-point manager - active cue-points might be filtered from payload